frontend/src/pages/Clusters/ClusterActions.js

```diff
@@ -88,8 +91,10 @@ export default function ClusterActions () {
     GetDcvSession(instance.instanceId, user, callback);
   }
 
-  return <div style={{marginLeft: "20px"}}>
-    <ClusterDeleteDialog clusterName={clusterName} />
+  return (<><div style={{marginLeft: "20px"}}>
```

Comment:
Because this component has a top-level `<div>` the `<>` and `</>` aren't needed here. If they're desired then the parenthesis can be removed and the div can be put on a newline (stylistically)